account_invoice_validation_queued/models/account_invoice.py

```diff
@@ -26,6 +26,9 @@ class AccountInvoice(models.Model):
         copy=False,
     )
 
+    def _get_validation_queued_identity_key(self, job):
+        return self.id
```

Comment:
The identity key should at least include the model and method names so 2 kind of jobs won't conflict, you can take example on https://github.com/OCA/queue/blob/fdcde048348694df37ce2f63686467cf762aeda5/queue_job/job.py#L102-L144